src/test/java/com/stripe/model/AccountTest.java

```diff
@@ -0,0 +1,116 @@
+package com.stripe;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.Account;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class AccountTest extends BaseStripeTest {
+	static String accountHash =
+		"{" +
+		"  \"id\": \"acct_1032D82eZvKYlo2C\"," +
+		"  \"email\": \"site@stripe.com\"," +
+		"  \"statement_descriptor\": null," +
+		"  \"display_name\": \"Stripe.com\"," +
+		"  \"timezone\": \"US/Pacific\"," +
+		"  \"details_submitted\": false," +
+		"  \"charges_enabled\": false," +
+		"  \"transfers_enabled\": false," +
+		"  \"currencies_supported\": [" +
+		"    \"usd\"," +
+		"    \"aud\"" +
+		"  ]," +
+		"  \"legal_entity\": {" +
+		"    \"type\": \"company\"," +
+		"    \"address\": {" +
+		"      \"city\": \"San Francisco\"" +
+		"    }" +
+		"  }," +
+		"  \"default_currency\": \"usd\"," +
+		"  \"country\": \"US\"," +
+		"  \"object\": \"account\"," +
+		"  \"business_name\": \"Stripe.com\"," +
+		"  \"business_url\": null," +
+		"  \"support_phone\": null," +
+		"  \"managed\": null" +
+		"}";
```

Comment:
Done. Also moved `resource` from `DeserializerTest` into `BaseStripeTest` and made `DeserializerTest` extend `BaseStripeTest`.